src/content/docs/apm/transactions/transaction-traces/transaction-traces-database-queries-page.mdx

```diff
@@ -31,69 +33,32 @@ If database query data is associated with your [selected transaction trace](/doc
 If you expect to see database data for a transaction trace but don't see it, you may need to change the [Database query settings](#settings).
 
 <img
-  title="APM-transaction-trace-database-queries.png"
-  alt="APM - Transaction trace - Database queries tab"
-  src={apmTransactionTraceDbQuery}
+    title="transaction trace database queries tab"
+    alt="A screenshot depicting the transaction trace database queries tab in the UI."
+    src={apmTransactionTraceDatabaseQueriesTab}
 />
-
 <figcaption>
   Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > APM & services > (select an app) > Monitor > Transactions > (select a transaction trace) > Database queries**.
 </figcaption>
 
 ## Use Database queries [#use]
 
-Use the **Database queries** page to analyze and troubleshoot a slow transaction. For example:
-
-1. You notice on the [APM **Overview** page](/docs/apm/applications-menu/monitoring/apm-overview-page) that a transaction is abnormally slow.
-2. You select a [transaction trace](/docs/apm/transactions/transaction-traces/transaction-traces) to get more details about that transaction.
-3. If the trace has associated database data, you select the **Database queries** page to examine the slow queries in the trace, and use them as the basis to [troubleshoot and improve your app's performance](#troubleshooting).
-
-Other functions include:
-
-<table>
-  <thead>
-    <tr>
-      <th style={{ width: "200px" }}>
-        If you want to...
-      </th>
-
-      <th>
-        Do this...
-      </th>
-    </tr>
-  </thead>
-
-  <tbody>
-    <tr>
-      <td>
-        Hide fast calls
-      </td>
-
-      <td>
-        By default, New Relic shows all queries, including fast calls (`averageDuration` less than five milliseconds). To hide these fast calls, select the **Hide fast calls** toggle.
+Here are situations where you can use the **Database queries** page to analyze and troubleshoot a slow transaction:
 
-        <img
-          title="Screenshot showing how to toggle to hide fast calls."
-          alt="Screenshot showing how to toggle to hide fast calls."
-          src={accountsFastCalls}
-        />
+* You notice on the [APM **Overview** page](/docs/apm/applications-menu/monitoring/apm-overview-page) that a transaction is abnormally slow
+* You select a [transaction trace](/docs/apm/transactions/transaction-traces/transaction-traces) to get more details about that transaction
+* You select the **Database queries** page to examine the slow queries in the trace, and use them as the basis to [troubleshoot and improve your app's performance](#troubleshooting)
 
+### View Stacktrace & Query
+When a stacktrace associated with a database query exists, click the row in the database queries table to view the formatted stacktrace.
```

Comment:
All references of stacktrace should be "stack trace"

Reply:
I forgot about the edits that were previously made to this doc. Thanks for flagging @ally-sassman